lib/geordi/settings.rb

```diff
@@ -1,83 +1,153 @@
-class GeordiSettings
-  require 'yaml'
+module Geordi
+  class Settings
+    require 'yaml'
 
-  GLOBAL_SETTINGS_FILE_NAME = '.config/geordi/global.yml'.freeze
-  LOCAL_SETTINGS_FILE_NAME = '.geordi.yml'.freeze
+    require File.expand_path('util', __dir__)
 
-  ALLOWED_GLOBAL_SETTINGS = %w[ pivotal_tracker_api_key ].freeze
-  ALLOWED_LOCAL_SETTINGS = %w[ headless_browser pivotal_tracker_project_id ].freeze
+    GLOBAL_SETTINGS_FILE_NAME = Util.testing? ? './tmp/global_settings.yml'.freeze : File.join(ENV['HOME'], '.config/geordi/global.yml').freeze
+    LOCAL_SETTINGS_FILE_NAME = Util.testing? ? './tmp/local_settings.yml'.freeze : './.geordi.yml'.freeze
 
-  def initialize
-    read_settings
-  end
+    GITPT_FILE_NAME = '.gitpt'.freeze
+    PROJECT_IDS_FILE_NAME = '.pt_project_id'.freeze
```

Comment:
Please inline these deprecated filenames (i.e. don't store them as constants). They will not be changed, but their functionality removed altogether at some point.